cdap-common/src/main/java/io/cdap/cdap/common/conf/Constants.java

```diff
@@ -1307,13 +1307,20 @@ public static final class AppMetadataStore {
      * Source Control Management metrics.
      */
     public static final class SourceControlManagement {
-
       public static final String CLONE_REPOSITORY_SIZE_BYTES =
           "source.control.git.repository.clone.size.bytes";
       public static final String CLONE_LATENCY_MS =
           "source.control.git.clone.latency.ms";
       public static final String COMMIT_PUSH_LATENCY_MILLIS =
           "source.control.git.commit.push.duration.ms";
+      public static final String PULL_COUNT =
+          "source.control.git.repository.pull.count";
+      public static final String PUSH_COUNT =
+          "source.control.git.repository.push.count";
+      public static final String PULL_PIPELINE_COUNT =
+          "source.control.git.repository.pull.pipeline.count";
```

Comment:
nit: not a strong opinion, but 'repository' seems a little redundant, does it make sense to remove it?